ui/src/pages/publications/[id]/versions/[versionId].tsx

```diff
@@ -90,17 +92,33 @@ export const getServerSideProps: Types.GetServerSideProps = async (context) => {
         api
             .get(`${Config.endpoints.publications}/${requestedId}/crosslinks?order=mix`, token)
             .then((res) => res.data)
-            .catch((error) => console.log(error))
+            .catch((error) => console.log(error)),
+        suggestedFromPublicationId
+            ? api
+                  .get(`${Config.endpoints.crosslinks}/${suggestedFromPublicationId},${requestedId}`, token)
+                  .then((res) => res.data)
+                  .catch((error) => console.log(error))
+            : Promise.resolve()
     ];
 
     const [
         { publicationVersion, versionRequestError },
         bookmarks = [],
         directLinks = { publication: null, linkedTo: [], linkedFrom: [] },
         flags = [],
-        crosslinks = []
+        crosslinks = [],
+        activeCrosslink = null
     ] = await Promise.all(promises);
 
+    let activeCrosslinkVote: Interfaces.CrosslinkVote | null = null;
+    if (suggestedFromPublicationId && activeCrosslink) {
+        activeCrosslinkVote =
+            (await api
```

Comment:
Why the mixture of async/await and promise chaining?

Reply:
Not sure. I think I just got confused :) will push it with just one of the two.